tools/generate_float_tests_header.py

```diff
@@ -525,6 +525,7 @@ def main():
 		write('#pragma once')
 		write('#include "settings.h"')
 		write('#include "../include/muu/integer_literals.h"')
+		write('#include "../include/muu/integer.h"')
```

Comment:
The changes to this file are unnecesasry; the code it generates doesn't use anything from your new header.